app/services/api/external-api/scenes/scene.ts

```diff
@@ -133,9 +135,9 @@ export class Scene {
   /**
    * removes all nodes from the scene
    */
-  clear(): void {
-    return this.scene.clear();
-  }
+  // clear(): void {
+  //   return this.scene.clear();
+  // }
 
```

Comment:
What is this commented out for?

Reply:
I commented it out to check that this method will be called from internal API and tests will be passed. There is no need for this now.